storage/storage-interface/src/lib.rs

```diff
@@ -478,6 +481,22 @@ pub trait DbReader: Send + Sync {
         unimplemented!()
     }
 
+    /// Gets a state value by state key, out of the ledger state indicated by the state
+    /// Merkle tree root with a sparse merkle proof proving state tree root.
```

Comment:
N00b question: is this comment accurate? We're not providing any proofs with this call, right?